google/cloud/servicedirectory_v1beta1/services/lookup_service/client.py

```diff
@@ -66,8 +67,38 @@ def get_transport_class(cls, label: str = None) -> Type[LookupServiceTransport]:
 class LookupServiceClient(metaclass=LookupServiceClientMeta):
     """Service Directory API for looking up service data at runtime."""
 
-    DEFAULT_OPTIONS = ClientOptions.ClientOptions(
-        api_endpoint="servicedirectory.googleapis.com"
+    @staticmethod
+    def _get_default_mtls_endpoint(api_endpoint):
```

Comment:
@arithmetic1728 Is pinning the minimum `google-api-core` version the only change I need to make to the library for mtls?

Reply:
yes